doc/sphinx/source/project/introduction.rst

```diff
@@ -4,176 +4,86 @@
 Introduction
 ############
 
-The nGraph Compiler stack provides industry-standard reference and implementation 
-guidelines for working with various :abbr:`Deep Learning (DL)` (DL) models and 
-optimizing an :abbr:`Artificial Neural Network (ANN)` (often abbreviated :term:`NN`) 
-to run graph-based computations for training, inference, testing, or validation.  
-Because today's NNs make use of many custom-purpose devices (FPGAs, GPUs, CPUs, 
-and custom silicon), having such a standard simplifies what would otherwise be 
-an enormously complex and difficult-to-scale pipeline (:ref:`Figure C <figure-C>`) 
-from "training with your favorite framework using GPUs" (:ref:`Figure D <figure-D>`), 
-to deploying that (now) pre-trained model in a datacenter or production 
-environment, where infrastructure owners or software developers renting anything 
-in a datacenter ought to be mutually concerned with **efficiency per-watt**, to 
-keep costs in check.
-
-So what exactly are the motivations behind the nGraph Compiler stack? 
+Accelerating deep learning performance is critical for further development of Artificial Intelligence (AI). Deep learning models & corresponding data set sizes are increasing rapidly, and deployment of scalable AI solution is challenging. Currently, there are two general approaches to advancing deep learning performance. First approach is to design hardware solutions dedicated to deep learning computation. Exisiting large silicon companies including Intel and deep learning hardware startups are actively developing ASICs (Application Specific Integrated Circuit) for speeding up deep learning performance for training & inference. 
+
+Second approach is to deliever performance via software optimization, and nGraph Compiler stack is our solution to the second apporach. We created nGraph compiler stack not only to provide software acceleration for the upcoming deep learning ASICs but also to unlock massive performance boost for existing hardware targets such as CPUs and GPUs. Additionally, it aims to address the scalability issue with kernel libraries which is a popular solution to accelerate deep learning performance.  
 
 Motivations
 ===========
 
-Kernel libraries do not support graph-level optimizations
----------------------------------------------------------
+The current state of art software solution for speeding up deep learning computation is to integrate kernel libraries such as Intel MKL-DNN and NVidia's CuDNN into deep learning frameworks. These kernel libraries offer runtime performance boost on specific hardware targets through highly optimized kernels and operator level optimizations.
```

Comment:
To achieve state-of-the-art performance with DL, the current solution is to integrate kernel libraries such as ...

Reply:
Everyone want to be an editor :) `Deep learning frameworks currently derive performance from kernel libraries such as...`